frontend/components/TableContainer/DataTable/StatusCell/_styles.scss

```diff
@@ -26,6 +26,16 @@
     background-color: $ui-offline;
   }
 
+  &--online,
+  &--offline {
+    .status-tooltip {
+      display: block;
+      white-space: normal;
+      max-width: 320px;
```

Comment:
308px max, padding is 6px on each side since we're going by styleguide here, might as well make it match